documentation/principles/empowering.md

```diff
@@ -2,12 +2,13 @@
 layout: sub-navigation
 order: 106
 title: Empowering
-excerpt: Short extract
+excerpt: We aim to allow teams to confidently deliver at pace and scale.
 tags:
 - principle
 eleventyNavigation:
   parent: principles
 ---
-Core Cloud aims to be light on process and empowering for teams.\
-Core Cloud allows team to engage at the right level for them; teams can create and manage their own resources or make use of our support model.\
-Core Cloud aims for 'Minimum Viable Platform' with no unnecessary abstractions or coupling.
\ No newline at end of file
+**We design for self-service.** \
+Teams should be able to provision their own resources from our service catalogue or make use of our support model and can pick which is right for them.
```

Comment:
```suggestion
Customers should be able to provision their own resources from our service catalogue or make use of our support model and can pick which is right for them.
```
Changed teams to customers but I think the `Empowering` principle needs a little more work but can't quite place what it should be saying right now. Mostly because we need to focus what these principles are for. i.e. if it's only for the core platform product or if its for the core cloud product family. I think principles should probably sit across all of the family

Reply:
I agree on both points, the theme I was going for was that we're not going to get in the way with process, only guardrails, if you can do that thing yourself do it - if not we're here.